hooks/terraform-fmt.sh

```diff
@@ -7,4 +7,8 @@ set -e
 # workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
 export PATH=$PATH:/usr/local/bin
 
-terraform fmt -recursive
+for file in "$@"; do
+  tmp_file="$(mktemp)"
+  terraform fmt - <"$file" >"$tmp_file"
```

Comment:
`-write=true` I think will do what you need

Reply:
Thanks. My problem was that the docs was saying the args are `DIR`, not `FILE` but it turns out it works fine with files.